pelican/tools/templates/tasks.py.jinja2

```diff
@@ -112,6 +112,9 @@ def livereload(c):
     for extension in static_file_extensions:
         static_file = '{0}/static/**/*{1}'.format(theme_path, extension)
         server.watch(static_file, lambda: build(c))
+    # Fire web browser at the URL
+    import webbrowser
+    webbrowser.open(f"localhost:{CONFIG['port']}")
```

Comment:
I don't think we have introduced F-strings into this file yet, so for now I think it's best to be consistent and use `.format()`, as seen in other parts of this file. Also, we should use the `CONFIG['host']` variable instead of hard-coding `localhost`. So perhaps something like:
```python
webbrowser.open("{host}:{port}".format(**CONFIG))
```

Reply:
I was worried if the host would be `0.0.0.0`, but this seems to work just fine on my GNU Linux + Firefox setup. Updated.